dist/routable-component.js

```diff
@@ -261,21 +261,21 @@ var routableComponent =
 	        }
 	      }
 	      delete this.components[name];
-	      this._listeners[name];
+	      delete this._listeners[name];
```

Comment:
__[eslint]__
* Error - Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
* Error - Expected indentation of 2 space characters but found 0. (indent)

<!-- Comment by [SideCI](https://sideci.com) -->